Automattic/woocommerce-payments
# Load woocommerce-subscriptions-core 1.3.0

Fixes #3538
Blocked by https://github.com/Automattic/woocommerce-subscriptions-core/pull/78

#### Changes proposed in this Pull Request

Load newer woocommerce-subscriptions-core's version, `1.3.0`.
Add changes of woocommerce-subscriptions-core `1.2.0` and `1.3.0` to the changelog.

#### Testing instructions

* Checkout this branch.
* [Husky hook should run `composer install`](https://github.com/Automattic/woocommerce-payments/blob/develop/.husky/post-checkout#L8) and your `vendor/woocommerce/subscriptions-core` should be updated to be `1.3.0`. Confirm that [latest changes from `1.3.0`](https://github.com/Automattic/woocommerce-subscriptions-core/commits/1.3.0) is present, for example, [vendor/woocommerce/subscriptions-core/changelog.txt is up-to-date](https://github.com/Automattic/woocommerce-subscriptions-core/commit/10555008c58cd08b3e3bd353b04de8dc55de475c#diff-b40cd67182487ef24807d9c9268329d35fbd96aa2b0a9cae69e2e0d746b1c666R3).

-------------------


**Post merge**
